.github/workflows/testing-msys2.yml

```diff
@@ -11,8 +11,9 @@ jobs:
     runs-on: windows-latest
 
     strategy:
+      fail-fast: false
       matrix:
-        msystem: [MSYS, MINGW32, MINGW64]
+        msystem: [MINGW32, MINGW64, CLANG32, CLANG64]
```

Comment:
I removed `MSYS` in ctags-win32 because it is not an environment for building Win32 apps.
However, I think we should keep this in this repository. (Even it is effectively the same as Cygwin.)

Reply:
Well, I will add it back after the issue is fixed.